flexget/plugins/input/kitsu.py

```diff
@@ -0,0 +1,114 @@
+from __future__ import unicode_literals, division, absolute_import
+import logging
+
+from flexget import plugin
+from flexget.config_schema import one_or_more
+from flexget.entry import Entry
+from flexget.event import event
+from flexget.utils.cached_input import cached
+from requests import RequestException
```

Comment:
Should fix your imports.

0. __future__ related stuff
1. builtin Python modules
2. third party libs
3. Flexget